README.md

```diff
@@ -17,13 +20,8 @@ The easiest way to run these components is with
 [Docker](https://www.docker.com/what-docker). To start these components,
 first install Docker for your platform and clone this repo.
 
-Next, create a `server/config.py` file based off the example at
-`server/config.py.example`. Pay special attention to the secret keys at the
-bottom of the example file. While other settings have defaults and can be
-omitted, these keys have no defaults and _must_ be provided to run the server.
```

Comment:
This removes the (previously inaccurate) statement that sample keys are not provided and must be input. I'm concerned however that people may deploy without understanding that there is an insecure key used if there is no mention of it. Thoughts?

Reply:
I moved it to a section further down titled: "Deploying to Any Non-Localhost Server" and tried to elaborate further on the reasoning for having to supply your own config.yaml (why it should not be in VCS, etc)

Reply:
Evidence for why I should complete a review before commenting. ;) That said, the statement is still inaccurate, no? I believe there are defaults and they need not be provided to run the server.